po/cs.po

```diff
@@ -2068,8 +2068,7 @@ msgstr "Délku předpony je třeba vyplnit"
 
 #: src/components/vm/snapshots/vmSnapshotsCard.jsx:236
 msgid ""
-"Previously taken snapshots allow you to revert to an earlier state if "
-"something goes wrong"
+"Snapshots are used to revert a VM to a earlier state"
```

Comment:
Please drop all the po changes. The original string changes, so we can't keep the translations as-is. Also, translations don't work that way, they are maintained in [weblate](https://translate.fedoraproject.org/projects/cockpit-machines/main/).